examples/ea4tx_remotebox.toml

```diff
@@ -25,3 +25,7 @@ name = "My EA4TX Remotebox"
 # switches with an index < 2 and before all other switched with an index > 2.
 index = 2
 portname = "/dev/ttyACM0"
+
```

Comment:
Hi Pablo, I think this empty line should be removed. The blank line makes the three parameters `ipaddress, ipport typeConnection` to be located in the root of the toml file. Howeever the parameters should be below the key `[mybandswitch]`.  This is likely the cause why cobra throughs a SIGSEGV when trying to access those parameters.

Reply:
Hi Tobi, it should work now!
Thanks for your help!